microscopy_SPIM001/samples.json

```diff
@@ -0,0 +1,14 @@
+{
+    "sample_id": {
+        "Description": "Sample ID"
+    },
+    "participant_id": {
+        "Description": "Participant ID from whom tissue samples have been acquired"
+    },
+    "sample_type": {
+        "Description": "Type of sample from ENCODE Biosample Type (https://www.encodeproject.org/profiles/biosample_type)"
+    },
+    "derived_from": {
+        "Description": "sample_id from which the sample is derived from"
+    }
+}
```

Comment:
`derived_from` is not used in the TSV file and should not appear in the json file